src/System.Text.Json/src/System/Text/Json/Serialization/Converters/DefaultConvertersOfTValue.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text.Json.Serialization.Policies;
+
+namespace System.Text.Json.Serialization.Converters
+{
+    internal static class DefaultConverters<TValue>
+    {
+        internal static readonly JsonValueConverter<TValue> s_converter = (JsonValueConverter<TValue>)DefaultConverters.Create(typeof(TValue));
```

Comment:
Why not to make it a method of `DefaultConverters`? I mean:

```csharp
internal static class DefaultConverters
{
    internal static JsonValueConverter<TValue> Get<TValue>() => Converter<TValue>.Instance;

    private static class Converter<TValue>
    {
        internal static readonly Instance = (JsonValueConverter<TValue>)Create(typeof(TValue));
    }
}
```

In that case there is no way to create an additional instance of the converter.